docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - .:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+      python cinema/management/commands/wait_for_db.py &&
+      python manage.py runserver 0.0.0.0:8000"
```

Comment:
The `wait_for_db.py` command should ideally be run before `python manage.py migrate` to ensure the database is ready before attempting migrations. Consider reordering these commands.